### ISSUE TITLE
Leaving a track doesn't reset everything like it warns

### ISSUE THREAD

**Issue by macta** [exercism/exercism.io#4182]
When you opt to leave a track, it warns you that your solutions will be deleted.

If you then join the track again - it prompts you with Congratulations, you've completed the track (assuming you did) - this means you can't start all over again.

I would expect it to also reset your progress state? (alternatively - give dev's a better way to test tracks and let them reset things themselves)

**Comment by jackhughesweb** [comment]
For reference: leaving a track currently will just archive the track

https://github.com/exercism/website/blob/e4623107e9e874cd8a1f432bd4260bcb96d82af7/app/controllers/my/user_tracks_controller.rb#L39

**Comment by macta** [comment]
The warning you get in the UI makes it sound pretty final... not archive.

Should you say- it will archive, but then what does that mean?

If I enjoyed doing the track or came back after 6 months having forgotten what I did - is there any way for me to start again?

I would think that leaving - getting a msg that it resets, and then coming back would let me start at the beginning again. It doesn't do this - when you come back it tells you congratulations - you are done?
